README.md

```diff
@@ -1,60 +1,55 @@
 
 ## Installation
-To install pylana, you need python version 3.6.0 or above. Pylana could be installed from pypi:
-```
-$ python3 -m pip install pylana
-```
-To establish a connection with the server, you could use either the _access token_ or the _API key_.
-```python
-import pylana
-myLana = pylana.LanaAPI(url = "https://cloud.lanalabs.com/", apikey = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9")
-```
 
-## Usage Example
-Pylana provides a simple interface which supports:
-* Uploading / retrieving / sharing event logs
-* Uploading / connecting target models
-* Creating / connecting / sharing shiny dashboards
- 
-__Upload an Event Log__ 
-```python
-# without case attribute
-myLana.uploadEventLog(logFile, logSemantics)
+You can install PyLana directly from PyPi with e.g. 
 
-# with case attributes
-myLana.uploadEventLogWithCaseAttributes(logFile, logSemantics, 
-    caseAttributeFile, caseAttributeSemantics)
+```bash
+$ pip install pylana
 ```
 
-__Retrieve Event Logs__
+# How to get started
+
+To connect with an api at e.g. 'https:://cloud-backend.lanalabs.com',first create an api with
+
 ```python
-# get a list of event logs associated with the current user
-myLana.getUserLogs()
+from pylana import create_api
 
-# get the log id of a particular log
-mylana.chooseLog(logName)
+api = create_api('https', 'cloud-backend.lanalabs.com', token)
 ```
 
-__Create a Shiny Dashboard__
+You will require a user token for this `token`. The returned api stores the url for a LANA Process Mining api as well as your authentication. After creation you can us it to manage the LANA process mining resources. Among other things you can upload data from python pandas data frames directly or connect logs and shiny dashboard resources referencing them by their names.
+
+To upload a new log called "new-event-log" and shiny-dashboard named "new-shiny-dashboard", and connect them with each other, you can use the following code
+
 ```python
-# initate a new shiny dashboard id
-id = mylana.createShinyDashboard(dashboardName)
+upload_response = api.upload_event_log_df(
+                            'new-event-log', df_event_log,
+                            time_format='YYYY-mm-dd',
+                            df_case=df_case_attributes)
+shiny_dashboard = api.create_shiny_dashboard('new-shiny-dashboard')
+connection_response = api.connect_shiny_dashboard(
+                                upload_response.json()['id'],
+                                shiny_dashboard['id'])
+```
 
-# upload the dashboard file to server
-mylana.uploadShinyDashboard(dashboardFile, id)
+We also provides basic methods to make HTTP verb requests directly to the LANA api endpoints. For example
 
-# connect the dashboard to an event log
-mylana.connectDashboardToLog(id, logId)
+```python
+response_list = api.get('/api/v2/dashboards')
 ```
 
+will return a response with a list of dashboard metadata.
+
 ## License
 [Apache License 2.0](http://www.apache.org/licenses/)
+
+## PyPI
```

Comment:
Should we add the directions from the Contributing markdown here?

Reply:
Oh, this is a residue. We can put directions here. The `CONTRIBUTING.md` is a kind-of standard, but it doesn't hurt to refer.